test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -819,7 +819,7 @@ def is_file(self, cpath, host):
         """
         Returns True if path exists otherwise false
         """
-        for _ in range(5):
+        for _ in range(20):
```

Comment:
Suggest you change this back to 5 attempts and add a time.sleep(0.5) after the check for rv. We don't need to continuously spin while waiting for the file to appear.